composer.json

```
@@ -23,10 +23,10 @@
   ],
   "minimum-stability": "dev",
   "require": {
-    "nosto/module-nostotagging": "^5.0.0",
+    "nosto/module-nostotagging": ">=5.0.0",
```

5.0 supports PHP 7.1, I don't think you could use typed properties back then.
Maybe change to 
```suggestion
    "nosto/module-nostotagging": "^6.0",
```